tests/php/src/Admin/OnboardingWizardSubmenuPageTest.php

```diff
@@ -45,77 +49,124 @@ class OnboardingWizardSubmenuPageTest extends WP_UnitTestCase {
 		set_current_screen( 'index.php' );
 
-		$this->assertEquals( 'Index - WordPress', $this->page->override_title( 'Index - WordPress' ) );
+		$this->assertEquals( 'Index - WordPress', $this->onboarding_wizard_submenu_page->override_title( 'Index - WordPress' ) );
 
-		set_current_screen( $this->page->screen_handle() );
+		set_current_screen( $this->onboarding_wizard_submenu_page->screen_handle() );
 
-		$this->assertEquals( 'AMP Onboarding Wizard - WordPress', $this->page->override_title( ' - WordPress' ) );
+		$this->assertEquals( 'AMP Onboarding Wizard - WordPress', $this->onboarding_wizard_submenu_page->override_title( ' - WordPress' ) );
 	}
 
 	/**
-	 * Tests OnboardingWizardSubmenuPage::render
+	 * Tests OnboardingWizardSubmenuPage::render()
 	 *
-	 * @covers ::render
+	 * @covers ::render()
 	 */
 	public function test_render() {
+		set_current_screen( 'admin_page_amp-onboarding-wizard' );
+
 		ob_start();
 
-		$this->page->render();
+		$this->onboarding_wizard_submenu_page->render();
 
 		$this->assertStringContains( '<div class="amp" id="amp-onboarding-wizard"></div>', ob_get_clean() );
 	}
 
 	/**
-	 * Tests OnboardingWizardSubmenuPage::screen_handle
+	 * Tests OnboardingWizardSubmenuPage::screen_handle()
 	 *
-	 * @covers ::screen_handle
+	 * @covers ::screen_handle()
 	 */
 	public function test_screen_handle() {
-		$this->assertEquals( $this->page->screen_handle(), 'admin_page_amp-onboarding-wizard' );
+		$this->assertEquals( $this->onboarding_wizard_submenu_page->screen_handle(), 'admin_page_amp-onboarding-wizard' );
 	}
 
 	/**
 	 * Tests OnboardingWizardSubmenuPage::enqueue_assets
 	 *
-	 * @covers ::enqueue_assets
+	 * @covers ::enqueue_assets()
 	 */
 	public function test_enqueue_assets() {
 		$handle = 'amp-onboarding-wizard';
 
-		$this->page->enqueue_assets( $this->page->screen_handle() );
+		$this->onboarding_wizard_submenu_page->enqueue_assets( $this->onboarding_wizard_submenu_page->screen_handle() );
 		$this->assertTrue( wp_script_is( $handle ) );
 		$this->assertTrue( wp_style_is( $handle ) );
 	}
+
+	/** @return array */
+	public function get_referrer_links() {
+		return [
+			'tools_page'        => [
+				static function () {
+					return admin_url( 'tools.php' );
+				},
+				true,
+			],
+			'amp_settings_page' => [
+				static function () {
+					return admin_url( 'admin.php?page=amp-options' );
+				},
+				true,
+			],
+			'login_page'        => [
+				'wp_login_url',
+				false,
+			],
+		];
+	}
+
+	/**
+	 * Tests OnboardingWizardSubmenuPage::get_close_link()
+	 *
+	 * @covers ::enqueue_assets()
+	 * @dataProvider get_referrer_links()
+	 *
+	 * @param callable $referrer_link_callback  Referrer link callback.
+	 * @param bool     $expected_referrer_close Whether the close link is expected to be the referrer.
+	 */
+	public function test_get_close_link( $referrer_link_callback, $expected_referrer_close ) {
+		$this->options_menu->add_menu_items();
+
+		$referrer                = $referrer_link_callback();
```

Comment:
For some reason calling `admin_url()` and/or `wp_login_url()` in the data provider caused an issue since it is before `setUp` and `is_ssl()` was not set properly.